api/Controllers/StaffController.cs

```diff
@@ -65,5 +69,26 @@ public IActionResult CheckCookie()
 
             return Ok("Authorized");
         }
+
+        /// <summary>
+        /// View all reservations that are for today or in the future.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet, Route("reservations")]
+        public async Task<IActionResult> GetReservations()
+        {
+            if (IsNotStaff(Request, out IActionResult? result))
+            {
+                return result!;
+            }
+
+            var reservations = await _reservationRepository.GetReservations();
+
+            // TODO: throws an error => Collection was modified; enumeration operation may not execute
+            // it might be useful to have a custom ITimeProvider
+            // var list = reservations.Where(r => r.Start >= DateTime.Today);
```

Comment:
At this point I kept getting the error `InvalidOperationException: Collection was modified; enumeration operation may not execute.` when calling the endpoint from the UI. I wasn't able to reproduce it through tests and it took me some time away.